src/components/formatters/include/formatters/formatter_json_rpc.h

```diff
@@ -278,11 +278,14 @@ int32_t FormatterJsonRpc::FromString(const std::string& str,
                                      ns_smart_objects::SmartObject& out) {
   int32_t result = kSuccess;
   try {
-    Json::Value root;
-    Json::Reader reader;
     namespace strings = ns_smart_device_link::ns_json_handler::strings;
+    Json::CharReaderBuilder builder;
+    const std::unique_ptr<Json::CharReader> reader(builder.newCharReader());
+    Json::Value root;
+    const int strLength = static_cast<int>(str.length());
 
-    if (false == reader.parse(str, root)) {
+    if (false ==
```

Comment:
@ydementieiev Can be replaced with if(!...) instead of explicit comparison with false?

Reply:
@AByzhynar done in [1345b2e](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/1345b2e4f52415e896a5fa3d232aa391c69751bb)